phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/ClasspathRepository.java

```diff
@@ -195,18 +203,15 @@ public boolean equals(Object o) {
             return false;
         }
 
-        ClasspathRepository that = (ClasspathRepository) o;
-
-        EqualsBuilder builder = new EqualsBuilder();
-
+        final ClasspathRepository that = (ClasspathRepository) o;
```

Comment:
I think on the contrary we should encourage small code cleaning like this without dedicated PR as long as it does not change the logic of the program, specially when it concerns classes that are modified inside the pull request